gbpservice/nfp/configurator/drivers/firewall/vyos/vyos_fw_driver.py

```diff
@@ -701,3 +701,24 @@ def delete_firewall(self, context, firewall, host):
             self._print_exception('Failure', resp.status_code, url,
                                   'create', resp.content)
             return const.STATUS_ERROR
+
+    def configure_healthmonitor(self, context, kwargs):
+        return self._check_vm_health(kwargs)
+
+    def _check_vm_health(self, kwargs):
```

Comment:
You can still use kwargs. The base class method takes kwargs as argument. Same code will work if we just have base class method.